### ISSUE TITLE
Consider providing stashgrep implementation

### ISSUE THREAD

**Issue by chrisc11** [mattkelly/stashr#1]
Users of this repo may also enjoy "stashgrep": https://gist.github.com/hartym/2584767  

Particularly useful when you are too lazy to clean out large git stash lists but remember some words you have in a stash somewhere

**Comment by mattkelly** [comment]
Good call! I always forget you can grep stashes. Would definitely save me some time.
